beast/physicsmodel/models.py

```diff
@@ -274,12 +290,20 @@ def make_seds(outname, specgrid, filters, av=[0., 5, 0.1], rv=[0., 5, 0.2],
     if fA is not None:
         fAs = np.arange(fA[0], fA[1] + 0.5 * fA[2], fA[2])
         g = creategrid.make_extinguished_grid(specgrid, filters, extLaw,
-                                              avs, rvs, fAs,
+                                              avs, 
+                                              rvs, 
+                                              fAs, 
+                                              av_prior_model,
+                                              rv_prior_model,
+                                              fA_prior_model,
                 add_spectral_properties_kwargs=add_spectral_properties_kwargs,
                                               absflux_cov=absflux_cov)
     else:
         g = creategrid.make_extinguished_grid(specgrid, filters, extLaw,
-                                              avs, rvs,
+                                              avs, 
+                                              rvs, 
+                                              av_prior_model,
+                                              rv_prior_model,
```

Comment:
Same as above -- pass the prior models as keywords: `av_prior_model=av_prior_model`, etc.

This is necessary in the `else` block, otherwise av_prior_model is passed as the 6th param (here, fAs).